expr/functions/groupByNode/function_test.go

```diff
@@ -50,6 +50,18 @@ func TestGroupByNode(t *testing.T) {
 				"qux": {types.MakeMetricData("qux", []float64{13, 15, 17, 19, 21}, 1, now32)},
 			},
 		},
+		{
+			Name:   "groupByNode_with_tag",
```

Comment:
Why this name ? Where the tags ?

Reply:
This test is about `groupByNode` arguments referring to tags rather than node indexes. See below:

```
groupByNode(metric1.foo.*.*,"name","sum")
```

That is telling `groupByNode` to group all the metrics with the same "name" tag (for this test, only one). From the [docs](https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.groupByNode) 

> Node may be an integer referencing a node in the series name or a string identifying a tag.